src/dialog/popover/popover.directive.spec.ts

```diff
@@ -6,7 +6,7 @@ import { By } from "@angular/platform-browser";
 import { NFormsModule } from "./../../forms/forms.module";
 import { IconModule } from "./../../icon/icon.module";
 import { StaticIconModule } from "./../../icon/static-icon.module";
-import { DialogPlaceholderService } from "./../dialog-placeholder.service";
+import { Placeholder } from "./../../placeholder/placeholder.service";
```

Comment:
Does this have to be
```suggestion
import { PlaceholderService } from "./../../placeholder/placeholder.service";
```